atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/cleaner/Scrubber.java

```diff
@@ -255,9 +256,8 @@ void runBackgroundScrubTask(final TransactionManager txManager) {
             final byte[] endRow = rangeBoundaries.get(i + 1);
             readerFutures.add(readerExec.submit(() -> {
                 BatchingVisitable<SortedMap<Long, Multimap<TableReference, Cell>>> scrubQueue = scrubberStore
-                        .getBatchingVisitableScrubQueue(batchSize, maxScrubTimestamp, startRow, endRow);
-                // Take one at a time since we already batched them together in KeyValueServiceScrubberStore.
-                BatchingVisitableView.of(scrubQueue).batchAccept(1, batch -> {
+                        .getBatchingVisitableScrubQueue(maxScrubTimestamp, startRow, endRow);
+                scrubQueue.batchAccept(batchSize, batch -> {
```

Comment:
Wrapping a BatchingVisitable in a batchingVisitableView just to call batchAccept on it is exactly like wrapping an Iterable in a FluentIterable just to call iterate on it. It accomplishes nothing.